### ISSUE TITLE
[question] What is "binary_cookie"

### ISSUE THREAD

**Issue by englercj** [kakaroto/SWProxy#62]
Just curious if in your exploration of the protocol you have determined what the "binary_cookie" field in the "HubUserLogin" request is. I'm curious because I don't recognize the encoding (alphanumeric + underscores) and it seems to change pretty often.

**Comment by kakaroto** [comment]
No idea, but since it's not needed to get monsters&runes information, there is no need to investigate it.

**Comment by englercj** [comment]
Well it is if you want to issue the request on behalf of the client, which is what I'm trying to do. I want to maintain a DB of my monsters outside the game, without having to configure proxy settings every single day to use a proxy system.

That's fine though, if you haven't looked into it I'll keep researching.

**Comment by lstern** [comment]
> Well it is if you want to issue the request on behalf of the client,

well, this is against TOS.

@kakaroto maybe we need to do an extra effort to better obfuscate the crypto key.

**Comment by kakaroto** [comment]
Yeah, that's against the TOS because it counts as "automation" which opens the door to bots and other similarly unfair usages. This tool only captures the data that the application itself sends.

@lstern I've obfuscated the key as best as I could, but it's just as easy to reverse it from our code than it is to reverse it from the original app. Either way, I've decided not to continue development of this tool because I've seen reports that com2us have said that this app is considered against ToS. So if the key is changed again, that will be the end of this little project unfortunately.
